tests/integration/test_fft_c2c.py

```diff
@@ -264,32 +164,17 @@ def check_3d_c2c(N, dtype=np.float64):
     assert allclose(Z, Z_num)
 
 
-def test_deferred_1d():
-    check_1d_c2c(N=10001)
-    check_1d_c2c(N=10001, dtype=np.float32)
-
-
-def test_deferred_2d():
-    check_2d_c2c(N=(128, 512))
-    check_2d_c2c(N=(128, 512), dtype=np.float32)
-
-
-def test_deferred_3d():
-    check_3d_c2c(N=(64, 40, 100))
```

Comment:
They are removed because they don't increase test coverage. All tests in the testing mode exercise the deferred path, and there's a separate mode for testing the eager path. Tests themselves don't need to be aware of what testing mode they run in. As far as I know, the FFT task is insensitive to the input size, so passing two different input sizes doesn't seem to add much other than testing cuFFT in different ways, which I believe should be done by the cuFFT test suite.

Reply:
I agree that cuFFT runs its own test cases for different sizes. However, numerical errors due to incorrect input parameters tend to become more evident with larger test cases. I question if the resources you are saving by not running these cases are worth it.

Reply:
In any case, I'll wait for an appropriate channel to discuss feedback.

Reply:
> However, numerical errors due to incorrect input parameters tend to become more evident with larger test cases

if there's a source of numerical errors other than cuFFT itself, I guess we could add back some of the tests with big inputs, but even then, I'd rather isolate them from functionality tests that exercise various parameters. And I don't think there's much of computation that could either worsen or mask the numerical errors from cuFFT.

Also, "numerical errors due to incorrect input parameters" don't make sense to me. If the input parameters are "incorrect", those should be caught rather than being passed to the rest of computation. (I suspect you were trying to say "suboptimal input parameters"?) If for some reason you want to do some additional numerical testing that the existing cuFFT test suite doesn't do, I guess it's fine to throw them here, but I feel those should rather be done to cuFFT itself...